api/integrations/github/models.py

```diff
@@ -47,3 +48,19 @@ class Meta:
                 name="unique_repository_data",
             )
         ]
+
+    @hook(BEFORE_DELETE)
+    def delete_feature_external_respurces(
+        self,
+    ) -> None:
+        from features.feature_external_resources.models import (
+            FeatureExternalResource,
+        )
+
+        FeatureExternalResource.objects.filter(
+            feature__in=self.project.features.all(),

```

Comment:
I'd avoid a `SELECT *` here:

```suggestion
            feature_id__in=self.project.features.values_list("id", flat=True),
```

Reply:
Done.